src/app/stake/page.tsx

```diff
@@ -0,0 +1,136 @@
+              <Table.ColumnHeaderCell className="flex w-1/4 p-2">
+                Asset
+              </Table.ColumnHeaderCell>
+              <Table.ColumnHeaderCell className="w-1/5 p-2 hidden md:block">
+                TVL
+              </Table.ColumnHeaderCell>
+              <Table.ColumnHeaderCell className="flex w-1/5 p-2">
+                Wallet Balance
+              </Table.ColumnHeaderCell>
+              <Table.ColumnHeaderCell className="w-1/5 p-2 hidden md:block">
+                Staked Balance
+              </Table.ColumnHeaderCell>
+              <Table.ColumnHeaderCell className="w-1/4 p-2 hidden lg:block">
+                Withdrawal Balance
+              </Table.ColumnHeaderCell>
+              <Table.ColumnHeaderCell className="flex w-1/4 p-2">
+                ACTIONS
+              </Table.ColumnHeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            {assets.map((asset) => (
+              <Table.Row
+                key={asset.assetSymbol}
+                className="flex items-center justify-between bg-white border-b hover:bg-gray-50 cursor-pointer"
+                onClick={handleOnClick(asset.assetSymbol)}
+              >
+                <Table.Cell className="w-1/4 p-2">
+                  <div className="flex ">
+                    <Image
+                      src={`/${asset.assetSymbol.toLocaleLowerCase()}.svg`}
+                      alt={`${asset.assetName} Logo`}
+                      width={0}
+                      height={0}
+                      className="size-[35px] rounded-full"
+                    />
+                    <span className="flex-col">
+                      <strong>{asset.assetSymbol}</strong>
+                      <br />
+                      <span className="text-xs">{asset.assetName}</span>
+                    </span>
+                  </div>
+                </Table.Cell>
+                <Table.Cell className="w-1/5 p-2 hidden md:block">
+                  {asset.totalBalance}
+                </Table.Cell>
+                <Table.Cell className="flex w-1/5 p-2">
+                  {asset.remainingBalance}
+                </Table.Cell>
+                <Table.Cell className="w-1/5 p-2 hidden md:block">
+                  {asset.remainingBalance}
+                </Table.Cell>
+                <Table.Cell className="w-1/5 p-2 hidden lg:block">
+                  {asset.remainingBalance}
+                </Table.Cell>
+                <Table.Cell className="flex w-1/4 p-2 flex gap-2">
+                  <button className="bg-[#A1FD59] text-black  ">
+                    <Image
+                      src="/download.svg"
+                      alt="Stake asset"
+                      width={29}
+                      height={32}
+                    />
+                  </button>
+                  <button className="bg-[#6D655D] text-white  ">
```

Comment:
same here. the text should not be #fff. It's isabelline. Nearly all of the colors are defined in the brand guide

Reply:
not referencing this anymore. using tailwind config file